### ISSUE TITLE
How to change default icinga host template

### ISSUE THREAD

**Issue by XnS** [theforeman/smart_proxy_monitoring#23]
Hi,

according to the documentation:
> In addition to the authentication, a Host template is required. By default, it uses "foreman-host" if none is provided at the Foreman WebUI

Where do I configure the default host template in the Foreman WebUI?

**Comment by bogenrs** [comment]
Hi,
is this possibly related to?:
https://github.com/theforeman/foreman_monitoring/issues/18

**Comment by timogoebel** [comment]
@bogenrs: Yes, I believe we have not implemented a setting to change the template name. Do you feel comfortable sending a pull request?
